src/main/java/bio/terra/service/dataset/AssetModelValidator.java

```diff
@@ -32,8 +32,6 @@ public boolean supports(@NotNull Class<?> clazz) {
     private void validateAssetName(String assetName, Errors errors) {
         if (assetName == null) {
             errors.rejectValue("name", "AssetNameMissing");
-        } else if (!ValidationUtils.isValidName(assetName)) {
-            errors.rejectValue("name", "AssetNameInvalid");
```

Comment:
There was no test covering this behavior. I can add one asserting we still see a `Pattern` validation error if reviewers would like.